.npmignore

```diff
@@ -0,0 +1,2 @@
+*
+!dist/*
```

Comment:
npmignore typically is a complete copy of gitignore, with additional lines added or removed.

presumably you want to include the LICENSE. also sometimes it's nice to include the `src` and `tsconfig.json`, and the docs.